vin/db/migrate/20150418030131_create_subscribers.rb

```diff
@@ -1,14 +0,0 @@
-class CreateSubscribers < ActiveRecord::Migration
-  def change
-    create_table :subscribers do |t|
-      t.string :name
-      t.string :email
-      t.string :phone
-      t.integer :address_id
-      t.string :fb
-      t.string :tw
-
-      t.timestamps
```

Comment:
Ahh, that's what I though. Rails does not make the `created_at` and `updated_at` attributes automatically. This does